app/models/manageiq/providers/kubevirt/infra_manager/provision_workflow.rb

```diff
@@ -25,8 +29,8 @@ def allowed_provision_types(_options = {})
     }
   end
 
-  def dialog_name_from_automate(message = 'get_dialog_name')
-    super(message, {'platform' => 'kubevirt'})
+  def dialog_name_from_automate(message = 'get_dialog_name', platform = 'kubevirt')
+    super(message, {'platform' => platform})
```

Comment:
changes to accomodate OSV provider https://github.com/ManageIQ/manageiq-providers-openshift/pull/266/files#diff-61a2477adaa497cdf4db91b97937275473566e67ae71343d74faab880e09b72fR7

Reply:
@nasark could you change the method definition to:
```
def dialog_name_from_automate(message, extra_attrs = {'platform' => 'kubevirt'})
```

That is how it is defined in the [parent class](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/infra_manager/provision_workflow.rb#L63)

Reply:
sounds good, updated the OSV PR as well